### ISSUE TITLE
Cocoapods

### ISSUE THREAD

**Issue by dvkch** [PSPDFKit-labs/PSTModernizer#1]
Hi,

I think adding this to Cocoapods would be nice. Here is a podspec file you could user : 

```
Pod::Spec.new do |s|
  s.name     = 'PSTModernizer'
  s.version  = '1.0'
  s.license  = 'MIT'
  s.summary  = 'Makes it easier to support older versions of iOS by fixing things and adding missing methods'
  s.homepage = 'https://github.com/PSPDFKit-labs/PSTModernizer'
  s.author   = { 'PSPDFKit Labs' => 'https://pspdfkit.com/' }
  s.source   = { :git => 'https://github.com/PSPDFKit-labs/PSTModernizer.git', :tag => s.version.to_s }
  s.source_files = 'PSTModernizer/*.{h,m,mm}'
  s.requires_arc = true

  s.xcconfig = { 'CLANG_MODULES_AUTOLINK' => 'YES' }
  s.ios.deployment_target = '6.0'
  s.tvos.deployment_target = '9.0'

end
```

The minimum iOS version could be adjusted, haven't been able to guess which one would be used. To support cocoapod versioning I usually tag each new version, and change the version number in the podspec.

To push to Cocoapods : 

```
# checks if the podfile is correct and the code compiles 
pod lib lint
# push to the public cocoapods repo
pod trunk push PSTModernizer.podspec
```

**Comment by zats** [comment]
I also thought about making each patch into a separate sub-spec. While it has certain maintenance overhead, it should allow to pick and choose which patches to ship with your app.
Also I haven't researched the topic, but I'm guessing it should be possible to get a default opt-in to every sub-spec behavior?
What do you think?

**Comment by lfarah** [comment]
This is fixed in #3

**Comment by elfenlaid** [comment]
Something odd has happened to spec, basically it has became unsearchable:

```
pod try PSTModernizer
Updating spec repositories
[!] Unable to find a specification for `PSTModernizer`
```

```
https://cocoapods.org/?q=PSTModernizer 
SORRY. NO PODS FOUND.
```
